files/systemd-sysusers-config/placeholders.conf

```diff
@@ -0,0 +1,18 @@
+# Reserved system users and groups for future use in Endless OS 7.
+#
+# This config defines system users and groups that have static IDs in Endless OS
+# 6, which may own files in /etc or /var of existing Endless OS systems, and so
+# the UIDs and GIDs need to be reserved in EOS7 too.
+#
+# System user config should be installed in the element for the corresponding
+# system service. The users below are special cases where we don't (yet) have
+# an element defined in EOS7.
+
+#Type Name       ID
+g     colord     112
+u!    colord     106:112
+g     kalite     128
+u!    kalite     121:128
+g     kolibri    139
+u!    kolibri    132:139
```

Comment:
kolibri comes from eos-kolibri https://github.com/endlessm/eos-kolibri/blob/Release_6.0.7/meson_options.txt#L36-L41
I can help add eos-kolibri.bst.

Reply:
#68 takes care this.

Reply:
I noticed a problem here introduced in #68, which i didnt' spot at review time: there are two sysusers.d files installed by this element (kolibri.conf and eos-kolibri.conf), so one gets ignored.

Reply:
https://github.com/endlessm/eos-build-meta/issues/71